packages/pagebuilder/lib/utils.js

```diff
@@ -173,3 +176,25 @@ export function getIsHidden(node) {
         isHidden: node.style.display === 'none'
     };
 }
+
+/**
+ * Modifies the node and children to include the current store view code in the
+ * link.
+ *
+ * @param node
+ * @returns node
+ */
+export function injectStoreCodeHref(node) {
+    if (process.env.USE_STORE_CODE_IN_URL === 'true') {
+        const storeViewCode =
+            storage.getItem('store_view_code') || STORE_VIEW_CODE;
+
+        // For each link update the href to include the active store code.
+        node.querySelectorAll('a').forEach(oldNode => {
+            const newNode = oldNode.cloneNode(true);
+            newNode.href = `/${storeViewCode}${oldNode.getAttribute('href')}`;
```

Comment:
@davemacaulay pointed out that the `href` may be relative or absolute so we need to make this a bit more resilient to the possibility of either.